30_ComputeShaderPathTracer/app_resources/common.glsl

```diff
@@ -4,7 +4,6 @@
 
 // basic settings
 #define MAX_DEPTH 3
```

Comment:
you can also make the depth configurable in ImGUI

Reply:
Done